src/test/lambda/explorer/standaloneNodes.test.ts

```diff
@@ -13,18 +13,27 @@ import { DefaultRegionNode } from '../../../lambda/explorer/defaultRegionNode'
 import { ErrorNode } from '../../../lambda/explorer/errorNode'
 import {
     DefaultStandaloneFunctionGroupNode,
-    DefaultStandaloneFunctionNode
+    DefaultStandaloneFunctionNode,
+    StandaloneFunctionNode
 } from '../../../lambda/explorer/standaloneNodes'
+import { CloudFormationClient } from '../../../shared/clients/cloudFormationClient'
+import { LambdaClient } from '../../../shared/clients/lambdaClient'
+import { ext } from '../../../shared/extensionGlobals'
 import { TestLogger } from '../../../shared/loggerUtils'
 import { RegionInfo } from '../../../shared/regions/regionInfo'
+import { MockLambdaClient } from '../../shared/clients/mockClients'
+
+async function* asyncGenerator<T>(items: T[]): AsyncIterableIterator<T> {
```

Comment:
We should probably move this to a shared util. Can you move it or add a TODO comment?